src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -87,7 +87,9 @@
                            [:master-run-dir "rundir"]
                            [:master-log-dir "logdir"]]]
       (if-let [value (get config setting)]
-        (.put puppet-config dir (ks/absolute-path value))))
+        (.put puppet-config dir (ks/absolute-path value)))
+      (if-let [value (get config :track-hiera-lookups)]
```

Comment:
Either the indentation is wrong or the nesting is. We don't want to do this in the `doseq` I assume?

Reply:
good catch here, I think this is why my test setup isn't working.